spec/browser/request.spec.js

```diff
@@ -126,6 +126,21 @@ describe('Request', () => {
       expect(path).toEqual('/api/example.json?user_email=email%2Btest%40example.com')
     })
 
+    it('encodes params if disableParamsEncode is not set', () => {
```

Comment:
Rebase, and put new specs in `src/request.spec.ts`

Reply:
Done: https://github.com/tulios/mappersmith/pull/251/commits/c07250dcc55de810a29ad3e0a56872ea82049cf5